modules/extensions-support/src/main/resources/META-INF/mule-extension-model.json

```diff
@@ -1825,6 +1825,561 @@
       ],
       "description": "Processes the nested list of message processors asynchronously using a thread pool",
       "modelProperties": {}
+    },
+    {
+      "name": "try",
+      "kind": "scope",
+      "blocking": true,
+      "executionType": "CPU_LITE",
```

Comment:
@dfeist can you check whether this makes sense? The try does build a chain with an intercepting processor that does not have a NB implementation. Can we be certain the other components are CPU_LITE?